tests/ids/add_ids.js

```diff
@@ -0,0 +1,54 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: autoid */
+
+'use strict';
+
+(function() {
+  bender.editor = {
+    config: {
+      enterMode: CKEDITOR.ENTER_P
+    }
+  };
+
+  bender.test({
+    setUp: function() {
+      this.editor = this.editorBot.editor;
+    },
+
+    'test it can add a unique id to a heading': function() {
+      var bot = this.editorBot,
+        editor = this.editor,
+        heading,
+        resumeAfter = bender.tools.resumeAfter,
+        startHtml = '<h1>This is a heading</h1>';
+
+      bot.setHtmlWithSelection(startHtml);
+
+      resumeAfter(editor, 'allIdsComplete', function() {
```

Comment:
I think this needs to go above line 25, otherwise you get a race condition where the event can fire before this is registered